Common/services/HTTPService.swift

```diff
@@ -57,13 +56,13 @@ class HTTPService {
             request.httpBody = httpBody.data(using: .utf8)
         }
 
-        let task = session.dataTask(with: request, completionHandler: {(data, response, error) in
+        let task = session.dataTask(with: request, completionHandler: {(data, _, error) in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/fd2a3b9c2c3f67178c28eea2e6e50998d8fb1238/Common/services/HTTPService.swift#L59"></span>Unused parameter “response” in a closure should be replaced with _.<br />
<code>unused_closure_parameter</code> <code>HTTPService.swift:59</code></td>
    </tr>
  </tbody>
</table>